pages/interface/components/Pagination/index.js

```diff
@@ -0,0 +1,44 @@
+import { Box, Link, Text } from '@/TabNewsUI';
+import { ChevronLeftIcon, ChevronRightIcon } from '@/TabNewsUI/icons';
+
+export default function Pagination({ pagination, paginationBasePath }) {
```

Comment:
Talvez essa refatoração tenha sido só um pretexto para adicionar os testes do front, mas que tal aproveitar para deixar a `Pagination` mais ou menos assim:

```suggestion
export default function Pagination({ previousPage, nextPage, basePath }) {
```

E pode ser usada assim:

```js
<Pagination {...pagination} basePath={paginationBasePath} />
```

Considerando que não existe página 0, então a alteração é simples, pois não há problema em verificar, por exemplo, `previousPage ?` no lugar de `pagination.previousPage ?`.